data/maps/BattleFrontier_BattleDomeBattleRoom/scripts.inc

```diff
@@ -5,17 +5,17 @@
 .set LOCALID_PLAYER, 13
 .set LOCALID_OPPONENT, 15
 
+.set NO_DRAW, 0
+.set DRAW_TRAINER, 1
+.set DRAW_TUCKER, 2
```

Comment:
Probably doesn't make sense to define assembly-time constants inside a labeled block? I moved them out and placed them with the other assembly-time constants.

Also, normalizing whitespace to always be a single space after the comma is very helpful for automated tools and arguably helps human readability and maintainability in most cases as well.